packages/common-server/src/analytics.ts

```diff
@@ -528,6 +528,11 @@ export class SegmentClient {
 // user props
 type UserProfileProps = {
   role?: string;
+  useCases?: string[];
+  useContext?: string;
```

Comment:
what do `useContext` and `publishingUseCase` represent here? maybe good to add some docstring comments for these.